scripts/geodata/osm/fetch_osm_address_data.sh

```diff
@@ -43,29 +43,22 @@ PREV_DIR=`pwd`
 cd $OUT_DIR
 
 # Download planet as PBF
-# TODO: currently uses single mirror, randomly choose one instead
 echo "Started OSM download: `date`"
 
 PLANET_PBF="planet-latest.osm.pbf"
-JAPAN_PBF="japan-latest.osm.pbf"
-
-wget --quiet http://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/pbf/planet-latest.osm.pbf -O $OUT_DIR/$PLANET_PBF &
-wget --quiet http://download.geofabrik.de/asia/japan-latest.osm.pbf -O $OUT_DIR/$JAPAN_PBF &
+wget --quiet https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf -O $OUT_DIR/$PLANET_PBF &
```

Comment:
Don't see mirrors anymore, load balancing might have been delegated to infrastructure provider